frigate/config.py

```diff
@@ -530,6 +530,7 @@ class CameraUiConfig(FrigateBaseModel):
 
 class CameraConfig(FrigateBaseModel):
     name: Optional[str] = Field(title="Camera name.", regex="^[a-zA-Z0-9_-]+$")
+    enabled: Optional[bool] = Field(default=True, title="Enable camera.")
```

Comment:
```suggestion
    enabled: bool = Field(default=True, title="Enable camera.")
```

You are using the Optional qualifier which means the field is nullable, but it shouldn't be. This will always have a default value of True or a user set value.

Reply:
Ok. Simply misunderstood the Optional.